allure-testng-adaptor/src/main/java/ru/yandex/qatools/allure/testng/AllureTestListener.java

```diff
@@ -124,6 +134,29 @@ public void onTestStart(ITestResult iTestResult) {
         getLifecycle().fire(event);
     }
 
+    private boolean isAtLeastJava8() {
+        try {
+            Class.forName("java.lang.reflect.Parameter");
```

Comment:
better to use `forName(String name, boolean initialize, ClassLoader loader)` method of `Class` and set the param `initialize` to `false`